core/src/main/java/com/novoda/noplayer/mediaplayer/AndroidMediaPlayerImpl.java

```diff
@@ -264,11 +267,37 @@ public void selectAudioTrack(PlayerAudioTrack audioTrack) {
     }
 
     @Override
+    public boolean hasAvailableSubtitles() {
+        NoPlayerLog.w("Subtitles not implemented for Android Media Player");
```

Comment:
I was kind off thinking to communicate to our library clients. Maybe it should throw ? As in is there a valid reason to support those methods being called ?